docs/source/system_reference_guide/oss_documentation/general_oss.ipynb

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
<p>I would change data to content in the bulleted list here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>